src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Ball();
+        Lottery ball2 = new Ball();
+        Lottery ball3 = new Ball();
+        ball.getRandomBall();
```

Comment:
The method 'getRandomBall()' is expected to return a value, but here it's not being used. You should store the result in a variable or use it directly if needed.